cmd/pebble/cmd_run.go

```diff
@@ -181,3 +185,23 @@ out:
 
 	return d.Stop(ch)
 }
+
+// ensurePebbleDir ensures that the pebble directory exists and is a directory.
+// If the directory doesn't exist, create it.
+func ensurePebbleDir(pebbleDir string) error {
```

Comment:
Wondering if this only makes sense for the default directory, regardless I think its fine.

Reply:
This code has gone now, I'm just using `os.MkdirAll`.